pages/linux/lvcreate.md

```diff
@@ -0,0 +1,19 @@
+# lvcreate
+
+> Creates a logical volume in an existing volume group.
```

Comment:
> Anyone who wants to use it must know what is a volume group due this is an advanced linux command mainly used by sysadmins or people who know what are doing.

I understand. But there is always a first time to learn everything. Think of the perspective from a newbie sysadmin who wants to learn to use the `lvcreate` command. He/she tries out tldr for that and then is stuck understanding what is a volume group. We always aim to give explanations and examples which allow the users to really understand the command, and not just make it a cheatsheet sort of thing.

> I think if I explain what is a volume group I need to explain first what is a logical volume and then it will not be a simplified man page.

Lets give it a shot, if that's okay with you. I will take a stab in shortening in to one line :smile: . If that does not seem possible, we won't go that deep.